src/psyclone/tests/psyGen_test.py

```diff
@@ -196,9 +196,26 @@ def apply(self, node, valid: bool = True):
 
     with pytest.raises(ValueError) as excinfo:
         trans.get_option("invalid")
-    assert ("'TestTrans' failed to get option 'invalid' as it is not "
-            "provided as a keyword argument to the apply method." in
-            str(excinfo.value))
+    assert ("option 'invalid' is not a valid option for 'TestTrans'. "
+            "Valid options are '['valid']." in str(excinfo.value))
+
+
+def test_transformation_apply_deprecation_message(capsys):
+    '''Test that passing the options dict to the Transformation.apply
+    function gets the expected deprecation message.'''
+    class TestTrans(Transformation):
+        '''Utility transformation to test methods.'''
+        def apply(self, node=None, options=None):
+            super().apply(node, options=options)
+
+    instance = TestTrans()
+    instance.apply(options={"dict": True})
+    out, err = capsys.readouterr()
+    assert ("PSyclone Deprecation Warning: The options parameters to "
+            "Transformation.apply and Transformation.validate are now "
+            "deprecated. Please use "
+            "the individual arguments, or unpack the options with "
+            "**options. See the documentation for more details." in out)
```

Comment:
s/documentation/Transformations section of the User Guide/

Reply:
Updated.